src/Components/Web.JS/src/Platform/Mono/MonoDebugger.ts

```diff
@@ -4,14 +4,11 @@ const currentBrowserIsChrome = (window as any).chrome
   && navigator.userAgent.indexOf('Edge') < 0; // Edge pretends to be Chrome
 
 
-let hasReferencedPdbs = false;
-
 export function hasDebuggingEnabled() {
-  return hasReferencedPdbs && currentBrowserIsChrome;
+  return currentBrowserIsChrome;
```

Comment:
Ah! I see why this would be the root cause of the user reported issue.

However, this is check is too liberal now and we'll end up running with debugging enabled if the user is running a publish app (which doesn't add any PDBs to the blazor.boot.json).

We'll probably need a more conservative check that accounts for the environment the app is running in and other properties in the boot.json.

I can make these modifications on top of your PR.

Thanks for looking at this!

Reply:
Thanks @captainsafia